src/modules/Navbar/Navbar.css

```diff
@@ -0,0 +1,89 @@
+.NavbarItems {
+    background: linear-gradient(90deg, rgb(0, 0, 0) 0%, rgb(6, 6, 7) 100%);
+    height: 100px;
```

Comment:
Not sure about hardcoded lengths.

Reply:
Not saying that you should change it, but in Android you want to use scaling units since hardcoded lengths do not scale on different sized screens.

Reply:
https://gist.github.com/basham/2175a16ab7c60ce8e001